docs/quick-start.md

```diff
@@ -0,0 +1,136 @@
+# DAG Factory Quick Start Guide
+
+**DAG Factory** is a Python library [Apache Airflow®](https://airflow.apache.org) that simplifies DAG creation using declarative YAML configuration files.
```

Comment:
```suggestion
**DAG Factory** is a Python library [Apache Airflow®](https://airflow.apache.org) that simplifies DAG creation using declarative YAML configuration files instead of Python.
```